lib/matplotlib/backends/backend_gtk3.py

```diff
@@ -366,7 +366,8 @@ def __init__(self, canvas, num):
             # all, so I am not sure how to catch it.  I am unhappy
             # doing a blanket catch here, but am not sure what a
             # better way is - JDH
-            verbose.report('Could not load matplotlib icon: %s' % sys.exc_info()[1])
+            log = logging.getLogger(__name__)
+            log.info('Could not load matplotlib icon: %s', sys.exc_info()[1])
```

Comment:
you can just do `log.info("Could not load matplotlib icon.", exc_info=True)` (magic! see https://docs.python.org/3/library/logging.html#logging.Logger.debug)